third_party/copy.bara.sky

```diff
@@ -55,7 +55,7 @@ core.workflow(
     ),
     destination = git.github_destination(
         url = moveUrl,
-        push = "aptos-main",
+        push = "main",
```

Comment:
Yes, we probably need an automation to sync e.g. nightly etc. PRs are nicer though because if there are crate dependencies, these do not automatically sync with copybara